pkg/config/config.go

```diff
@@ -287,6 +287,7 @@ type Config struct {
 	Graveler struct {
 		EnsureReadableRootNamespace bool `mapstructure:"ensure_readable_root_namespace"`
 		BatchDBIOTransactionMarkers bool `mapstructure:"batch_dbio_transaction_markers"`
+		TriggerDeleteSensorAt       int  `mapstructure:"trigger_delete_sensor_at"`
```

Comment:
I don't know what this parameter does.  Can you document it, or perhaps add units (does `num_deletes_to_trigger_delete_sensor` make sense?), or change the name?